powerhome/playbook
# Fix missing CSS rule on radio kit

#### Screens

Before:

![2021-01-13 16 39 38](https://user-images.githubusercontent.com/508128/104505739-23a60a80-55c3-11eb-9d34-cf720f2c8753.gif)

After:

![2021-01-13 17 03 03](https://user-images.githubusercontent.com/508128/104505747-286abe80-55c3-11eb-91b4-709c14766de9.gif)

#### Breaking Changes

No.

#### Runway Ticket URL

None.

#### How to test this
{{
This only happens in Pulse (but there is no instance of the Radio kit in master yet) or on a brand new react app using playbook.

I've created [this repo](https://github.com/drborges/playbook-ui-spikes) that one can use to reproduce and verify the fix.

#### Checklist:


*The normal release cut off deadline is 3p EDT each week. Please reach out to the release team if you have an urgent request or need a release off cycle.*
